### ISSUE TITLE
Lots of small files that leads to out of inodes on filesystem

### ISSUE THREAD

**Issue by m8-t** [dbeaver/cloudbeaver#1366]
Hey there,

I am using cloudbeaver 22.2.5 in kubernetes. Recently I tried to do an export of data but was not able to download the data.
Further investigations lead me to a "No space left on device" error in the logs.
I found a folder which held alot of small files and a `df -i` showed that all inodes are used.

Folder is `/opt/cloudbeaver/workspace/.metadata/.http-sessions`

I deleted most of the files as I didn't find any reason to keep them.
They just contain some java stuff I do not understand (not a developer).

For example:

```
cat 0__0.0.0.0_1trbem5y5ec3vudatgasfwwl485846
1trbem5y5ec3vudatgasfwwl4858460.0.0.0)<)<)<srjava.lang.Integer⠤8Ivaluexrjava.lang.Number
```

What are these files for and how can I stop cloudbeaver from using up all the inodes?

Thanks

**Comment by kseniaguzeeva** [comment]
Thank you for the interest in CloudBeaver. The files can be useful in some cases. In any case we are going to find a solution to the issue you described above.

**Comment by dariamarutkina** [comment]
Hello @dan-m8t,
Fixed in milestone 23.0.1:
Added setting MaxInactiveInterval value in SessionHandler when initializing Jetty server. The value is taken from the server configuration in seconds
```
server: {
  expireSessionAfterPeriod: 1800000
}
```
This instructs the server to delete expired sessions files.
Thank you for the interest in CloudBeaver.

**Comment by nicon89** [comment]
I have same issue on version 23.3.0, even though I've set expireSessionAfterPeriod to 3600 seconds I'm getting all inodes empty after few days. What else can I do?

**Comment by nicon89** [comment]
Inode usage metric:
![image](https://github.com/dbeaver/cloudbeaver/assets/9640905/c62f4218-5694-404f-83c1-3d1bb30780a9)

**Comment by dariamarutkina** [comment]
Hello, @dan-m8t!
We fixed it in milestone 24.0.2. Please take a look at the new version of CloudBeaver 24.0.2
Thank you for your interest in CloudBeaver.

**Comment by nicon89** [comment]
Thank you @dariamarutkina for the update.
When can I expect [cloudbeaver-ee](https://hub.docker.com/r/dbeaver/cloudbeaver-ee/tags) image to be updated?
